{
	"schema_version": "1.6.0",
	"id": "MAL-2024-6185",
	"modified": "2024-10-24T01:01:59Z",
	"published": "2024-06-25T13:43:57Z",
	"summary": "Malicious code in urlkib3 (PyPI)",
	"details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
	"affected": [
		{
			"package": {
				"ecosystem": "PyPI",
				"name": "urlkib3",
				"purl": "pkg:pypi/urlkib3"
			},
			"versions": [
				"1.27.15"
			],
			"database_specific": {
				"source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/urlkib3/MAL-2024-6185.json"
			}
		}
	],
	"references": [
		{
			"type": "ARTICLE",
			"url": "https://www.reversinglabs.com/blog/beware-impostor-http-libraries-lurk-on-pypi"
		}
	],
	"credits": [
		{
			"name": "ReversingLabs",
			"contact": [
				"https://www.reversinglabs.com"
			],
			"type": "FINDER"
		}
	],
	"database_specific": {
		"malicious-packages-origins": [
			{
				"id": "RLMA-2024-04989",
				"import_time": "2024-06-28T02:51:06.252554546Z",
				"modified_time": "2024-06-25T13:43:57Z",
				"sha256": "99cc280691c2ee2fbb120fb81188117203bbeadaf825f5c5dc08edfbaa138614",
				"source": "reversing-labs",
				"versions": [
					"1.27.15"
				]
			},
			{
				"id": "RLUA-2024-09471",
				"import_time": "2024-10-24T01:00:23.490518337Z",
				"modified_time": "2024-10-16T14:53:26Z",
				"sha256": "ab827b42395ec50c08b64a97cb089b539bd1525ab8a22449fc91b7f75815aa3e",
				"source": "reversing-labs"
			}
		]
	}
}
